recipes/nerval/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "nerval" %}
+{% set version = "1.0.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 67630089ab868c13682226455a26672ee8d236e0bc3d5e6b0d2a260114007ebb
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - pandas
+    - numpy
+    - matplotlib

```

Comment:
Can you just confirm that you indeed need the full `matplotlib` package that includes Qt and not just the `matplotlib-base`?